### PR TITLE
Minor UI tweaks: 

### PR DESCRIPTION
**Pull request by singalen**
* focus and Enter action in a couple of dialogs
* drive sort menu shortcuts

### CODE REVIEW

File: src/ncwin.cpp [hunk 1]
```diff
@@ -534,20 +534,20 @@ NCWin::NCWin()
...
 	_mdLeft.AddSub( _LT( "&Sort mode" ) , &_mdLeftSort );
 	_mdLeft.AddCmd( ID_DEV_SELECT_LEFT, _LT( "Change &drive" ),  "Shift-F1" );
 
-	_mdRightSort.AddCmd( ID_SORT_BY_NAME_R,  _LT( "SM>Name", "&Name" ) );
-	_mdRightSort.AddCmd( ID_SORT_BY_EXT_R,  _LT( "SM>&xtension", "&Extension" ) );
-	_mdRightSort.AddCmd( ID_SORT_BY_MODIF_R, _LT( "SM>Modif. Time", "Modif. &Time" ) );
-	_mdRightSort.AddCmd( ID_SORT_BY_SIZE_R,  _LT( "SM>Size", "&Size" ) );
-	_mdRightSort.AddCmd( ID_UNSORT_R, _LT( "SM>Unsorted", "&Unsorted" ) );
+	_mdRightSort.AddCmd( ID_SORT_BY_NAME_R,  _LT( "SM>Name", "&Name" ), "Ctrl-F3" );
+	_mdRightSort.AddCmd( ID_SORT_BY_EXT_R,  _LT( "SM>&Extension", "&Extension" ), "Ctrl-F4" );
```
Comment (helgp) [inline src/ncwin.cpp]: Ampersend in "SM>&Extension" does not follow pattern in other lines of the block. Typo?

Comment (singalen) [inline src/ncwin.cpp]: Indeed, it's a string id.

Comment (singalen) [inline src/ncwin.cpp]: Re-fixed in master.

Comment (helgp) [inline src/ncwin.cpp]: Far has Russian shortcuts.

Comment (singalen) [inline src/ncwin.cpp]: OK, I might do it in the next UI tweaks portion.

**Pull request merged**